### PR TITLE
Grammar fix for README

### PR DESCRIPTION
**Pull request by fredster33**
No issue applicable.

Changes proposed in this pull request:
-Minor grammar fix for README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,11 +30,11 @@ Therefore, we prefer to focus on its increment instead of a total overturn.
 
 Apache ShardingSphere begins to focus on pluggable architecture from version 5.x, features can be embedded into project flexibility.
 Currently, the features such as data sharding, replica query, data encrypt, shadow test, and SQL dialects / database protocols such as MySQL, PostgreSQL, SQLServer, Oracle supported are all weaved by plugins.
-Developers can customize their own ShardingSphere systems just like building lego blocks. There are lots of SPI extensions for Apache ShardingSphere now and increasing continuously.
+Developers can customize their own ShardingSphere systems just like building lego blocks. There are lots of SPI extensions for Apache ShardingSphere now and they are increasing continuously.
 
 ShardingSphere became an [Apache](https://apache.org/index.html#projects-list) Top Level Project on April 16, 2020.
 
-Welcome communicate with community via [mail list](mailto:dev@shardingsphere.apache.org).
+You can communicate with community via the [mailing list](mailto:dev@shardingsphere.apache.org).
```
Comment (tristaZero) [inline README.md:37]: How about "Welcome to community"?

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @fredster33 ,
Thank for your fixing, could you look at the comment?

File: README.md (hunk 1 above)
Comment (fredster33) [reply README.md:37]: Hi there, could you please clarify?

Comment (fredster33) [reply README.md:37]: I think "You are welcome to communicate" would work

Comment (tristaZero) [reply README.md:37]: Nice, go ahead.

**Review by tristaZero — CHANGES_REQUESTED** [review]

**Comment by tristaZero** [comment]
Hi @fredster33 ?

**Comment by fredster33** [comment]
All right, I've made the changes. The checks are currently running. Sorry for being late!

**Review by tristaZero — APPROVED** [review]
Thanks @fredster33

**Pull request merged**